SAP/fundamental-react
# feat: update fundamental-styles to 0.7.0-rc.13

### Description

Modal
   - renamed `Dialog`
   - removed `closeProps`
   - `actions` now required
Button
   - underlying class name changes
ButtonGroup
   - classname changed to `fd-segmented-button`
- Tab
    - remove `disabled` prop
   - change underlying html - added `span` around tab text
- Shellbar
    - logo `<a>` is now `<span>`
    - added `<div>` around Identifier component
- Alert
    - renamed MessageStrip
    - classname changes 
    - removed Icon from underlying html
- Dropdown
    - renamed `Select`
    - remove `standard` prop
- ListGroup
    - renamed “List”
    - classname changes
    - so many changes! basically a new component!
- MultiInput
    - underlying html changes
    - `Popover` body now built with `List` component
    - lots of underlying html changes for validation states
    - `state` prop replaced with `validationState` - takes object
    - `localizedText` prop removed
- Combobox Input
    - underlying html changes
    - `menu` prop renamed `list` - now takes `List` component
    - `state` prop now `validationState` - takes an object 
- FormMessage
    - now a `<div>` from `<span>`
- InputGroup
    - `state` prop now `validationState` - takes an object
    - underlying html changes to include `FormMessage` when passed `validationState`
- SearchInput
    - `state` prop now `validationState` - takes an object
    - underlying html changes to include `FormMessage` when passed `validationState`https://github.com/SAP/fundamental-react/pull/885
- Toggle
    - renamed to “Switch”
    - removed size options, now use `compact` property
    - `labelProps` no longer needed, additional props will be spread to the `<label>` element
    - underlying html changes - no longer wrapped  in `FormItem`
* Checkbox
    * new `children` prop
    * `value` prop changed
    * underlying html has changed significantly 
* FormRadioItem
    * underlying html has changed significantly 
* FormMessage
    * now an internal component
* FormInput
    * removed `state` property
    * new `validationState` property
* FormItem
    * remove `isHorizontal` prop
    * remove `isInline` prop (was internal only)
* FormRadioGroup
    * now outputs `<div>` instead of Fragment
* FormLabel
    * remove `isCheckbox` prop
    * remove `isRadio` prop
    - removed `isToggle` prop


See https://github.com/SAP/fundamental-react/pull/888, https://github.com/SAP/fundamental-react/pull/885 for a divided pr

Known Issues:
Toggle/Switch
        - internalLabel prop not working in prop descriptions
        - not an accessible pattern
        - this is going to change ASAP as it is currently being redesigned 
        - icons are misaligned inside switch
Token: 
        - selected state - did not implement - what is this even used for?
Shellbar
        - `fd-shellbar__button` needs more specificity
  ListText
        - `—no-wrap` class only works in selects - this should probably be part of select, not list
Select
        - compact not working because `—fd-forms-height-compact` is not part of our postcss pipeline anymore - just SAP variables: https://github.com/SAP/fundamental-styles/pull/711
MultiInput
        - `fd-tokenizer__input` does not take priority over `fd-input` - getting the fd-input outline box (needs more specificity)
Validation states
        - validation state styling - FS has the validation message as a list item instead of a div:https://github.com/SAP/fundamental-styles/issues/712
        - “onClickOutside” is not being handled correctly
